openstack/networking/v2/ports/types.go

```diff
@@ -0,0 +1,10 @@
+package ports
+
+type PortStatus string
+
+const (
+	PortStatusActive PortStatus = "ACTIVE"
+	PortStatusBuild  PortStatus = "BUILD"
+	PortStatusDown   PortStatus = "DOWN"
+	PortStatusError  PortStatus = "ERROR"
```

Comment:
Given the package name "ports", the identifiers for these constants can be just "StatusActive" without fear of confusion.